.rustfmt.toml

```diff
@@ -2,4 +2,4 @@ tab_spaces = 2
 edition="2021"
 imports_layout="HorizontalVertical"
 imports_granularity="Crate"
-reorder_imports=true
```

Comment:
We can just rely on the default value for this:

https://rust-lang.github.io/rustfmt/?version=v1.5.1&search=import#reorder_imports